packages/app/src/app/overmind/onInitialize.ts

```diff
@@ -104,4 +106,26 @@ export const onInitialize: OnInitialize = async (
   });
 
   effects.preview.initialize(overmindInstance.reaction);
+
+  // show terms message on first visit since new terms
+  if (!effects.browser.storage.get(seenTermsKey)) {
+    effects.notificationToast.add({
+      message:
+        'Hello, our privacy policy has been updated recently. What’s new? CodeSandbox emails. Please read and reach out.',
+      title: 'Updated Privacy',
+      status: convertTypeToStatus('notice'),
+      sticky: true,
+      actions: {
+        primary: [
+          {
+            label: 'Open Privacy Policy',
+            run: () => {
+              window.open('https://codesandbox.io/legal/privacy', '_blank');
+            },
+          },
+        ],
+      },
+    });
+    effects.browser.storage.set(seenTermsKey, true);
+  }
```

Comment:
Use early return instead

```suggestion
  if (effects.browser.storage.get(seenTermsKey)) {
    return;
  }
  
  effects.notificationToast.add({
    message:
      'Hello, our privacy policy has been updated recently. What’s new? CodeSandbox emails. Please read and reach out.',
    title: 'Updated Privacy',
    status: convertTypeToStatus('notice'),
    sticky: true,
    actions: {
      primary: [
        {
          label: 'Open Privacy Policy',
          run: () => window.open('https://codesandbox.io/legal/privacy', '_blank'),
        },
      ],
    },
  });
  effects.browser.storage.set(seenTermsKey, true);
```